### PR TITLE
Pal pause

### PR DESCRIPTION
**Pull request by marijnvdwerf**
Still needs some cleanup of names.

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 93 functions (+0.92%) on `pal`.

**Review by ethteck — APPROVED** [review]
looks great! just if you could fix the 8-space-tabbed arrays like gPauseBadgesElements_fr, but otherwise, awesome shtuff and thank you

**Comment by marijnvdwerf** [comment]
The way I understand it, the game uses `gPauseBadgesElements` as a nested array with `gCurrentLanguage` used to pick the first level. We should look if we can use `#define gCurrentLanguage 0` for the other releases, to see if we can use the same data structure. That'd make `ifdef`'ing the other languages easier.

How would those things need to be formatted as a nested array? Just a clump of items (as they are now), nested like this?
```c
HudScript* gPauseBadgesElements[4][22] = {
    [LANGUAGE_EN] = {
        &HES_StatBp, &HES_StatBp, &HES_StatBp, &HES_StatBp,
        &HES_StatBp, &HES_StatBp, &HES_StatBp, &HES_StatBp,
        &HES_StatBp, &HES_StatBp, &HES_StatBp, &HES_StatBp,
        &HES_StatBp, &HES_StatBp, &HES_StatBp, &HES_StatBp,
        &HES_StatBp, &HES_ListPrevPage, &HES_ListNextPage,
        &HES_Dash, &HES_LabelBpNeeded, &HES_StatusStar1
    },
    [LANGUAGE_DE] = {
        /* ... */
    },
    [LANGUAGE_FR] = {
        /* ... */
    },
    [LANGUAGE_ES] = {
        /* ... */
    },
};
```

**Comment by ethteck** [comment]
yep! looks great

File: src/pause/pause_stats.c [hunk 1]
```diff
@@ -81,19 +86,34 @@ s8 gPauseStatsGridData[] = {
     3, 9,
     3, 10
 };
+
+#if VERSION_PAL
+#define VAR_1 122
```
Comment (HailSanta) [inline src/pause/pause_stats.c:91]: VAR_1 = STAR_POWER_X
VAR_2 = COLLECTABLES_X

File: src/pause/pause_tabs.c [hunk 2]
```diff
@@ -159,7 +208,7 @@ void pause_tabs_draw_stats(MenuPanel* menu, s32 baseX, s32 baseY, s32 width, s32
     hud_element_draw_without_clipping(gPauseTabsIconIDs[0]);
     if (gPauseMenuCurrentTab == 0) {
         if (gPauseTabsWindowIDs[menu->col] == 25) {
-            pause_set_cursor_pos(gPauseTabsWindowIDs[menu->col], baseX, baseY + 6);
+            pause_set_cursor_pos(gPauseTabsWindowIDs[menu->col], baseX + CURSOR_OFFSET_X, baseY + 6);
```
Comment (HailSanta) [inline src/pause/pause_tabs.c:211]: more specifically, TABS_CURSOR_OFFSET_X

File: src/pause/pause_map.c [hunk 3]
```diff
@@ -1,6 +1,14 @@
 #include "pause_common.h"
 #include "message_ids.h"
 
+#if VERSION_PAL
+#define VAR_1 36
```
Comment (HailSanta) [inline src/pause/pause_map.c:5]: VAR_1 = MAP_TITLE_X
VAR_2 = MAP_TITLE_WIDTH

**Comment by ethteck** [comment]
@marijnvdwerf are the remaining things ones we should tackle now or is it better to just make a gh issue and merge this? I'm happy with it as-is I think.

**Comment by marijnvdwerf** [comment]
@ethteck Since everything builds, I think it's _technically_ fine to merge. The `#define LANGUAGE_EN (0)` stuff is a bit weird though, so I imagine you'd want a plan for that before merging it in.

**Comment by marijnvdwerf** [comment]
@ethteck Should be good now

**Pull request merged**